### PR TITLE
SB-135 Implement a cron task for remote repository index downloading 

### PR DESCRIPTION
**Pull request by kate-novik**

### CODE REVIEW

**Comment by carlspring** [comment]
Looks good! Is this ready to be merged?

**Comment by kate-novik** [comment]
No, test is sometimes fail. I think, it happens, because we create twice
Indexing context (when repository start and when we call function
download() ). I'll check one idea about this case tomorrow and write you.

25 дек. 2016 г. 10:15 ПП пользователь "Martin Todorov" <
notifications@github.com> написал:

File: strongbox-storage/strongbox-storage-indexing/pom.xml [hunk 1]
```diff
@@ -244,9 +244,21 @@
...
+            <artifactId>artifact-downloader</artifactId>
+            <version>1.0-SNAPSHOT</version>
+        </dependency>
+
+        <dependency>
             <groupId>junit</groupId>
             <artifactId>junit</artifactId>
         </dependency>
+
+        <dependency>
```
Comment (carlspring) [inline strongbox-storage/strongbox-storage-indexing/pom.xml]: Do we need this dependency? Isn't it coming as a dependency of the `artifact-downloader`? (Just asking)...

Comment (kate-novik) [inline strongbox-storage/strongbox-storage-indexing/pom.xml]: Yes, of course. I checked something and forgot delete.

**Comment by carlspring** [comment]
@cs-devops-bot rebuild

**Comment by carlspring** [comment]
@cs-devops-bot rebuild

**Comment by carlspring** [comment]
This will be handled as part of #205.

**Pull request closed without merging**